datasetinsights/configs/faster_rcnn_fine_tune.yaml

```diff
@@ -42,3 +42,8 @@ optimizer:
 pretrained: False
 pretrained_backbone: True
 synchronize_metrics: True
+mlflow:
+  experiment_name: datasetinsights
+  run_name:
+  tracking_client_id:
+  tracking_host_id:
```

Comment:
since these are inside the mlflow block, we can simplify the names to improve readability. For example:
```yaml
experiment:
run:
client_id:
host:
```